Corona-Warn-App/src/main/res/layout/include_target.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
maybe call it `include_submission_target_selection` as target is really generic and I think this include is very specific.